MIT.md

```diff
@@ -0,0 +1,2 @@
+GIRMA TAREKEGN 
+CopyRight @2022
```

Comment:
- [ ] This is your license file so please take it seriously kindly add the whole statement in your MIT.md file using this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :+1:
- [ ] After adding these contents kindly add your **name**, **current year** and **app type** in the template at repective places :+1: